backend/fms_core/templates.py

```diff
@@ -631,9 +632,9 @@
   "user prefill info": {
       "QC Date (YYYY-MM-DD)": "date",
       "Volume Used (uL)": "number",
-      "Quality Instrument": "qc-instrument",
+      "Quality Instrument": SAMPLE_QC_QUALITY_INSTRUMENTS,
       "Quality Flag": VALID_QC_FLAG_CHOICES,
-      "Quantity Instrument": "qc-instrument",
+      "Quantity Instrument": SAMPLE_QC_QUALITY_INSTRUMENTS,
```

Comment:
The list for quantity and quality is not the same. Please use the Sample QC template as reference.